tests/TestHelpers/WebDavHelper.php

```diff
@@ -898,4 +878,66 @@ public static function getMtimeOfResource(
 		$mtime = new DateTime($xmlPart[0]->__toString());
 		return $mtime->format('U');
 	}
+
+	/**
+	 *
+	 * @param string $property
+	 * @param string $baseUrl
+	 * @param string $user
+	 * @param string $password
+	 * @param string $path
+	 * @param string|null $xRequestId
+	 * @param int|null $davPathVersionToUse
+	 *
+	 * @return string
+	 * @throws GuzzleException
+	 */
+	public static function getFilePropertyForPath(
```

Comment:
what was wrong in the previous function and creating new function since the implementation looks same to me?

Reply:
its more dynamic then before can use for all type of property rather than fileid

Reply:
if so then may be only change the function name no need to create new function.

Reply:
not in this PR